frontend/src/store/actions/notifications.js

```diff
@@ -0,0 +1,5 @@
+export const types = {
+  FETCH_INIT: 'FETCH_INIT',
+  FETCH_SUCCESS: 'FETCH_SUCCESS',
+  FETCH_FAILURE: 'FETCH_FAILURE',
```

Comment:
@thadk would it be ok if we replace `FETCH` by `NOTIFICATIONS` here?

Reply:
This makes more sense in Redux! Thanks.